### PR TITLE
Expose devtools plugin permissions through the metadata endpoint

### PR DESCRIPTION
**Pull request by brianphillips**
## Hey, I just made a Pull Request!

Add missing permissions to be exposed in metadata endpoint for devtools plugin

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/fast-zebras-watch.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-devtools-backend': patch
+'@backstage/plugin-devtools-common': patch
+---
+
+Expose permissions through the metadata endpoint.
```
Comment (vinzscam) [inline .changeset/fast-zebras-watch.md:6]: same as https://github.com/backstage/backstage/pull/17845/files#r1197023656 😄

**Review by vinzscam — COMMENTED** [review]
looks good! 🙏

**Review by awanlin — APPROVED** [review]
Thanks for this @brianphillips! 🚀

**Comment by brianphillips** [comment]
> Feel like I totally missed that was needed now

@awanlin - The only place I found it referenced was in [the "diff" of code changes required when adding permissions to a plugin](https://backstage.io/docs/permissions/plugin-authors/02-adding-a-basic-permission-check#authorizing-using-the-new-permission).

**Review by jhaals — APPROVED** [review]
Looking good 👍

**Pull request merged**